nexus-sdk/src/main/java/io/nexusrpc/handler/OperationContext.java

```diff
@@ -21,6 +22,7 @@ public static Builder newBuilder(OperationContext context) {
   private final Map<String, String> headers;
   // This is not included in equals, hashCode, or toString
   private final @Nullable OperationMethodCanceller methodCanceller;
+  private final List<Link> links = new ArrayList<>();
 
   private OperationContext(
```

Comment:
One thing I was debating internally was if we should add a static `OperationContext.current()` or something to get the current context?

@bergundy @cretz thoughts?

Reply:
Sure, SGTM. Wondering if we also want to pass it in explicitly to handlers in that case.

Reply:
We talked offline, I forgot we made the context optional. Definitely support adding the static `current()` method.

Reply:
Scratch that, it was a misunderstanding.

Reply:
I'd say let's wait until needed. Since it's made available on the handler anyways probably no need (if a user wants to set it in a thread local for its downstream utilities, no prob). And if there is enough desire, we can add it.

Reply:
I think we should wait until people really ask for it since it's available as a parameter and users can set it as thread local if that's how they prefer to propagate the context.

Reply:
We may have uses for it from the Temporal SDK depending on how we build out the workflow helpers like `signal`, fine to punt for now.